AccessibilityTests/Sources/AccessibilityTests.swift

```diff
@@ -92,7 +91,9 @@ final class AccessibilityTests: XCTestCase {
     private static let notHumanReadableMessage = "Label not human-readable"
     
     /// Use this array to filter add specific filters to ignore specific issues for certain elements
-    private static let ignoredA11yIdentifiers: [String: [FilterType]] = [A11yIdentifiers.authenticationStartScreen.appVersion: [.auditType(.hitRegion)]]
+    private static let ignoredA11yIdentifiers: [String: [FilterType]] = [A11yIdentifiers.authenticationStartScreen.appVersion: [.auditType(.hitRegion)],
+                                                                         A11yIdentifiers.pollView.summary: [.auditType(.hitRegion)],
+                                                                         A11yIdentifiers.serverConfirmationScreen.serverPicker: [.compactDescription(notHumanReadableMessage)]]
```

Comment:
Super minor nit just for readability could we do it like this?

```suggestion
    private static let ignoredA11yIdentifiers: [String: [FilterType]] = [
        A11yIdentifiers.authenticationStartScreen.appVersion: [.auditType(.hitRegion)],
        A11yIdentifiers.pollView.summary: [.auditType(.hitRegion)],
        A11yIdentifiers.serverConfirmationScreen.serverPicker: [.compactDescription(notHumanReadableMessage)]
    ]
```